exception.S

```diff
@@ -4,8 +4,10 @@
 .align 1
 .globl __mtvec_handler
 
-.globl decode_illegal
-.type  decode_illegal, @function
+.globl handle_exception
```

Comment:
This also looks extra.

Reply:
Do you know why it was there originally? To my knowledge neither the original or new form of this would _no anything_, but I kept it assuming there was some intent I didn't understand. Should I just remove it?

Reply:
I thought you added it, didn't you?

Reply:
Ah I see what you mean. Yeah I don't think the .globl did anything here before, but also `handle_exception` doesn't exist so there's no reason to have it now.

Reply:
Good points on both; removed.